XCUITests/FxScreenGraph.swift

```diff
@@ -187,16 +213,26 @@ func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozill
 
 extension Navigator {
     // Open a URL. Will use/re-use the first BrowserTab or NewTabScreen it comes to.
-    func openURL(urlString: String) {
+    func openURL(urlString: String, regularBrowsing: Bool = true) {
```

Comment:
I considered this; I think for the meantime, let's not worry about changing the `openURL` method at all.

The reason is this: if you change `regularBrowsing = false`, then despite you calling `openURL(urlString: url, regularBrowsing=false)` then it is very easy to get into the state where you are not in private browsing, even though you've called a method which says it'll get you to private browsing.

You will be able to make this guarantee for `openNewURL` however.